### PR TITLE
added check isEmpty

### PR DESCRIPTION
**Pull request by Aliaksei-Halkin**

### CODE REVIEW

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/impl/CandidateEventServiceImpl.java [hunk 1]
```diff
@@ -79,9 +79,13 @@ public void delete(Long id) {
 
     @Override
     public Page<ResponseCandidateEventDto> getAllPageable(Pageable pageable, String search) {
-        Node rootNode = new RSQLParser().parse(search);
-        Specification<CandidateEventEntity> spec = rootNode.accept(new CustomRsqlVisitor<>());
-        return candidateEventDAO.findAll(spec,pageable)
-                .map(candidateEventMapper::convertEntityToDto);
+        if(search.isEmpty()) {
```
Comment (akarzhaneuski) [inline sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/impl/CandidateEventServiceImpl.java:82]: Bad code formatting => bad readability. Please reformat code with Ctrl + Alt + L  hothey

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/impl/InterviewFeedbackServiceImpl.java [hunk 2]
```diff
@@ -105,11 +105,13 @@ public void delete(Long id) {
 
     @Override
     public Page<ResponseInterviewFeedbackDto> getAllPageable(Pageable pageable, String search) {
-        Node rootNode = new RSQLParser().parse(search);
-        Specification<InterviewFeedbackEntity> spec = rootNode.accept(new CustomRsqlVisitor<>());
-        return interviewFeedbackDAO.findAll(spec, pageable)
-                .map(interviewFeedbackMapper::convertEntityToDto);
+        if(search.isEmpty()){
```
Comment (akarzhaneuski) [inline sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/impl/InterviewFeedbackServiceImpl.java:108]: And here please

**Review by akarzhaneuski — CHANGES_REQUESTED** [review]

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/impl/CandidateEventServiceImpl.java (hunk 1 above)
Comment (Aliaksei-Halkin) [reply sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/impl/CandidateEventServiceImpl.java]: I forgot to do it( Done!

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/impl/InterviewFeedbackServiceImpl.java (hunk 2 above)
Comment (Aliaksei-Halkin) [reply sandbox-backend/src/main/java/com/exadel/team2/sandbox/service/impl/InterviewFeedbackServiceImpl.java]: done

**Review by akarzhaneuski — APPROVED** [review]

**Pull request merged**